daemon/api/api.yaml

```diff
@@ -1664,6 +1664,226 @@ paths:
         500:
           $ref: '#/components/responses/500'
 
+  /node/name/{nodename}/system/disk:
+    get:
+      description: View the disk
+      operationId: GetNodeSystemDisk
+      parameters:
+        - $ref: '#/components/parameters/inPathNodeName'
+      responses:
```

Comment:
Missing: for all added pathes
      security:
        - basicAuth: []
        - bearerAuth: []